### PR TITLE
added a location endpoint

### PR DESCRIPTION
**Pull request by tdevinda**
I need a back end location endpoint. So I added it.
If it pleases my liege @keshan, please pull this. .env will be sent via email

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -138,6 +150,13 @@ def get_needs_by_location():
 def ping():
     return "pong"
 
+@app.route("/api/v1/location", methods=["GET"])
+def getLocationFromString():
+    parameters = {'input': request.args.get('address'), 'inputtype': 'textquery', 'key': GOOGLE_API_KEY, 'fields': 'geometry'}
+    response = requests.get('https://maps.googleapis.com/maps/api/place/findplacefromtext/json', params=parameters)
+    if response.status_code == 200:
+        return json.dumps(response.json()), 200, {'Content-Type': 'application/json'}
```
Comment (keshan) [inline app.py]: Shall we use jsonify instead of json.dumps as it consistent with rest of the API.

**Review by keshan — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (tdevinda) [reply app.py]: Will push the changes. Give me some time.

Comment (tdevinda) [reply app.py]: Replaced with jsonify. Please check now.

**Comment by keshan** [comment]
LGTM, Thanks!!

**Pull request merged**